src/main/java/scala/compat/java8/ScalaStreaming.java

```diff
@@ -0,0 +1,905 @@
+package scala.compat.java8;
+
+import scala.compat.java8.converterImpl.*;
+import scala.compat.java8.collectionImpl.*;
+import java.util.stream.*;
+import scala.compat.java8.runtime.CollectionInternals;
+
+public class ScalaStreaming {
```

Comment:
Here's my take on it: https://github.com/szeiger/scala-java8-compat/commit/cd37db5bb1ba2041fa01c7233452201bc4f4cfee